### ISSUE TITLE
Deploy enum support to crates.io

### ISSUE THREAD

**Issue by brylee10** [petar-dambovaliev/serde_alias#3]
Thanks for the helpful proc macro. Could the most recent version of this macro which gives support for using the macro on enums be pushed to crates.io? 

Related to the thread here: https://github.com/petar-dambovaliev/serde_alias/issues/1#issuecomment-1775055494

**Comment by jilvin** [comment]
Yes please. I couldn't find any support for enums anywhere other than implementing them manually.

Enum support is not available on `serde-aux` either for case-insensitive matching at the moment.
